DataFormats/GEMDigi/interface/ME0PadDigi.h

```diff
@@ -0,0 +1,37 @@
+#ifndef DataFormats_GEMDigi_ME0PadDigi_h
+#define DataFormats_GEMDigi_ME0PadDigi_h
+
+/** \class ME0PadDigi
+ *
+ * Digi for ME0 trigger pads
+ *  
+ * \author Sven Dildick
+ *
+ */
+
+#include <boost/cstdint.hpp>
+#include <iosfwd>
+
+class ME0PadDigi{
+
+public:
+  explicit ME0PadDigi (int pad, int bx);
+  ME0PadDigi ();
+
+  bool operator==(const ME0PadDigi& digi) const;
+  bool operator!=(const ME0PadDigi& digi) const;
+  bool operator<(const ME0PadDigi& digi) const;
+
+  // return the pad number. counts from 1.
+  int pad() const { return pad_; }
+  int bx() const { return bx_; }
+
+private:
+  uint16_t pad_;
+  int32_t  bx_; 
```

Comment:
if bx_ is represented as a int16_t the size of this class (and similar ones above) is 50% less.